packages/cli-plugin-metro/src/commands/start/index.ts

```diff
@@ -88,6 +88,10 @@ export default {
       name: '--no-interactive',
       description: 'Disables interactive mode',
     },
+    {
+      name: '--terminal',
+      description: 'Specify terminal to use for running Metro',
```

Comment:
~~Can we update this description like so,~~ (**update**: see top-level feedback, let's relocate this) and demonstrate the bail-on-port-exists behaviour in the test plan?

```suggestion
      description: 'When this option is provided, starts the dev server in a separate terminal window using the provided terminal app path. In this mode, the command will immediately bail if another process is running on the same port.',
```